great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -23,6 +25,13 @@
     cast,
 )
 
+try:
+    from typing import Literal  # type: ignore[attr-defined]
+except ImportError:
+    # Fallback for python < 3.8
+    from typing_extensions import Literal  # type: ignore[misc]
```

Comment:
Is this preferred over checking for version using `version` library?

Reply:
This is how `mypy` suggests doing it...

```python
import sys
if sys.version_info >= (3, 8):
    from typing import Literal
else:
    from typing_extensions import Literal
```

Although personally I don't love having to import `sys`

https://mypy.readthedocs.io/en/stable/runtime_troubles.html?highlight=typing_extensions#using-new-additions-to-the-typing-module

Reply:
I'm good either way!

Reply:
I'm fine either way. `try/except` is less specific and will cause an extra import for the users who are <3.8. We also use `from packaging import version` in various places throughout the codebase.